src/dispatch.c

```diff
@@ -122,9 +122,9 @@ STATIC_ASSERT(sizeof(long) >= sizeof(void*), pointer_is_bigger_than_long);
 
 qd_error_t qd_dispatch_load_config(qd_dispatch_t *qd, const char *config_path)
 {
-    qd->dl_handle = dlopen(QPID_DISPATCH_LIB, RTLD_LAZY | RTLD_NOLOAD);
+    qd->dl_handle = dlopen(NULL, RTLD_LAZY | RTLD_NOLOAD);
```

Comment:
This dlopen is done for backward compatibility, right?  Can you drop a comment here explaining that?

Reply:
I was thinking whether I could move the `dlopen`  to the python side, in `configure_dispatch`, but I was not exactly sure I can make the things work exactly the way they are now.